plugin-relay/src/server.rs

```diff
@@ -68,20 +71,38 @@ impl AlumetPlugin for RelayServerPlugin {
     }
 
     fn init(config: ConfigTable) -> anyhow::Result<Box<Self>> {
-        let config = deserialize_config(config)?;
+        let mut config: Config = deserialize_config(config)?;
+
         Ok(Box::new(RelayServerPlugin { config }))
     }
 
     fn start(&mut self, alumet: &mut AlumetStart) -> anyhow::Result<()> {
-        let addr: SocketAddr = match self.config.ipv4_only {
-            true => SocketAddr::V4(SocketAddrV4::new(Ipv4Addr::LOCALHOST, self.config.port)),
-            false => SocketAddr::V6(SocketAddrV6::new(
-                Ipv6Addr::LOCALHOST,
+        // Retrieve the ip from the config using
+        // [the trait ToSocketAddrs](https://doc.rust-lang.org/stable/std/net/trait.ToSocketAddrs.html)
+        // of (String, u16) to use a raw address without any port.
+        let current_ip = match (self.config.address.clone(), self.config.port)
+            .to_socket_addrs()
+            .context("Parsing of the `address` param of the server relay plugin")?
```

Comment:
Please include the config value in the error message. You can omit "server relay plugin", it's included by the logger based on the source file.

Reply:
Here is the error with the context:

```
 Caused by:
        0: Parsing of the `address` param of the server relay plugin: 127.0.0.1:50051
        1: failed to lookup address information: Name or service not known
```